worldguard-legacy/src/main/java/com/sk89q/worldguard/protection/flags/DefaultFlag.java

```diff
@@ -179,4 +189,36 @@ private DefaultFlag() {
 
         return null;
     }
+
+    /**
+     * Add a new custom flag to the flags list
+     *
+     * @param flag The flag to add
+     */
+    static void addFlag(Flag<?> flag) {
```

Comment:
Try to keep the default-visible things at a minimum.
This can just as well be invoked by using new CustomFlagBrokerImpl().addFlag(...), so hiding it by making it default-visible makes no sense anyway.